doc/conf.py

```diff
@@ -14,6 +14,12 @@
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.insert(0, os.path.abspath('.'))
 
+# -- Setup environment variables for elephant --------------------------------
+# set the following environment variables to ensure execution of plots when
+# building docs. This is of importance in connection with elephant modules
+# using OpenCL (pyopencl) or CUDA (pycuda).
+
+os.environ["ELEPHANT_USE_OPENCL"] = "0"
```

Comment:
I think you also need
```
os.environ["ELEPHANT_USE_CUDA"] = "0"
```

Reply:
thanks, added with eb9ff3804bfbff6ecc4bdaf923d04b114fdfef61